lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -39,6 +39,8 @@ def call
             licence_transaction_will_continue_on: details["will_continue_on"].presence,
             licence_transaction_continuation_link: details["continuation_link"].presence,
             licence_transaction_licence_identifier: licence_identifier(details),
+            primary_publishing_organisation: tagging["primary_publishing_organisation"] || ::LicenceTransaction.schema_organisations.first,
```

Comment:
I'd probably want validation for a missing primary publishing organisation, rather than automatically using the default org but this can be addressed at a later point as it isn't a concern of this PR 👍

Reply:
Agreed - we have the ticket for actually doing the import - it'll build on this I guess, and add proper validation of primary publishing org in the CsvTaggingValidator class.